primitives/src/lib.rs

```diff
@@ -50,8 +50,8 @@ pub use units::amount::{Amount, SignedAmount};
 #[doc(inline)]
 pub use units::{
     block::{BlockHeight, BlockInterval},
-    fee_rate::FeeRate,
-    weight::Weight,
+    fee_rate::{self, FeeRate},
```

Comment:
I"ll explain the reasoning behind and you can comment on whether or not you think its valid.

High level:
The aim is that users of `bitcoin` should not need to "know about" `primitives` or `units`. Also users of `primitives` should not need `units`. By "know about" I mean never explicitly have them in import statements. To achieve that aim I was trying to make all export paths the same i.e., `bitcoin::FeeRate` is the same as `primitives::FeeRate` is the same as `units::FeeRate`. 

This case:
Some folk copy what we do in `bitcoin` and use qualified paths in there import statements. Without `self` one can write `units::fee_rate::FeeRate` but cannot write `primitives::fee_rate::FeeRate`.

Reply:
I see, thanks for the explanation.  Though I think logically, FeeRate is in units, and I think you introduce more complexity here by trying to mask that.  Is this to prevent a breaking change?  I guess before all the crate smashing this used to be `bitcoin::FeeRate`?  Personally I'd prefer a more minimal approach and just use only `units::FeeRate` going forward but that's just me.

Reply:
Its exactly because of the subjective nature of imports that we trying to accommodate multiple different ways of doing things. And if we are going to do so we should do so fully and correctly (IMO).